cmd/justification/main.go

```diff
@@ -69,7 +70,13 @@ func realMain(ctx context.Context) error {
 		return fmt.Errorf("failed to setup grpc auth handler: %w", err)
 	}
 
-	p := justification.NewProcessor(kmsClient, cfg, authHandler)
+	firestoreClient, err := firestore.NewClient(ctx, cfg.FirestoreProjectID)
+	if err != nil {
+		return fmt.Errorf("failed to create Firestore client: %w", err)
+	}
+
+	fireStoreRemoteConfig := config.NewFirestoreRemoteConfig(firestoreClient, "JVS/JustificationConfig")
```

Comment:
I checked the best practise [here](https://cloud.google.com/firestore/docs/best-practices#field_names) It does not list the naming convention and I tested the collection name as "JVS"  and document id as "JustificationConfig", it works.  I didn't make it a constant here because the firestore implementation remote config is to read/write config from the specific firestore document. It is possible that one api may need to read/write multiple configs from different firestore documents.

Reply:
In some exmaples, I saw they are using lower case. I will change it to lower case.